docs/man_pages/lib-management/library.md

```diff
@@ -7,11 +7,13 @@ General | `$ tns library <Command>`
 
 Lets you manage third-party native libraries in your project. You must run the `library` command with a command extension.
 
+IMPORTANT: The `tns library` set of commands are deprecated and will be removed in a future release. Use the `tns plugin` set of commands instead. For more information, <% if(isHtml) { %>see the [plugin](plugin.html) set of commands.<% } %><% if(isConsole) { %>run `tns help plugin`.<% } %>.
```

Comment:
Hmmm, make this "The `tns library` command is deprecated" perhaps - sorry about that.